Apps/WebClient/src/NewClientApp/src/stores/covidLaboratory.ts

```diff
@@ -0,0 +1,146 @@
+﻿import { defineStore } from "pinia";
+import { Covid19TestResultState } from "@/models/datasetState";
+import { LoadStatus } from "@/models/storeOperations";
+import { SERVICE_IDENTIFIER } from "@/ioc/identifier";
+import { ILaboratoryService, ILogger } from "@/services/interfaces";
+import { container } from "@/ioc/container";
+import { ref } from "vue";
+import {
+    Covid19LaboratoryOrder,
+    Covid19LaboratoryOrderResult,
+} from "@/models/laboratory";
+import { DatasetMapUtils } from "@/stores/utils/DatasetMapUtils";
+import { useErrorStore } from "@/stores/error";
+import { ErrorSourceType, ErrorType } from "@/constants/errorType";
+import { ResultError } from "@/models/errors";
+import RequestResult from "@/models/requestResult";
+import { ResultType } from "@/constants/resulttype";
+import { ActionType } from "@/constants/actionType";
+
+export const defaultCovid19TestResultState: Covid19TestResultState = {
+    data: [],
+    status: LoadStatus.NONE,
+    statusMessage: "",
+    error: undefined,
+};
+
+export const useLaboratoryStore = defineStore("laboratory", () => {
```

Comment:
```suggestion
export const useCovid19TestResultStore = defineStore("covid19TestResult", () => {
```

Reply:
this is a cloning error, I copied the whole file due to the similarities